web/src/components/InputElement.vue

```diff
@@ -39,6 +42,10 @@ const component = computed(() => {
       return DateInput
     case InputType.CHECKBOX:
       return CheckboxInput
+    case InputType.MULTITEXT:
```

Comment:
You could easily argue for `chips` instead of Multitext. Although the component only supports `string[]` right now and we'll probably need a dedicated component for multiple dates soon. Right now, the idea is to user MultiText / MultiDate/ Multi... for InputType & Components and extract a `useChips` composable for the multiComponents...

(However, we should probably talk to design and create a shared language)

Reply:
decided  for chips (@l30ni3)